include/nccl_ofi_platform.h

```diff
@@ -12,6 +12,17 @@ extern "C" {
 #include <rdma/fabric.h>
 #include <rdma/fi_endpoint.h>
 
+struct ec2_platform_data {
```

Comment:
These changes can't go in `nccl_ofi_platform.h`, because this file has to work on platforms which are not AWS specific.